aws/dual-region/kubernetes/region0/camunda-values-failover.yml

```diff
@@ -48,4 +43,12 @@ zeebe:
       fieldRef:
         fieldPath: metadata.namespace
   - name: ZEEBE_BROKER_CLUSTER_MEMBERSHIP_PROBETIMEOUT
-    value: "100ms"
+    value: "500ms"
+  - name: ZEEBE_BROKER_CLUSTER_MEMBERSHIP_PROBEINTERVAL
```

Comment:
do we need these values in the failover if they are already defined in the base?

Reply:
that's why I don't like the failover helm values.
You can't just overwrite a single env var in an overlay, so you have to define everything twice.